tracer/src/Datadog.Trace/Activity/Handlers/DefaultActivityHandler.cs

```diff
@@ -102,18 +108,26 @@ public void ActivityStarted<T>(string sourceName, T activity)
                     return;
                 }
 
-                ActivityMappingById.GetOrAdd(activity.Id, _ => new(activity.Instance, CreateScopeFromActivity(activity, parent, traceId, spanId, rawTraceId, rawSpanId)));
+                Log.Information("GetOrAdd for {Name} with ID: {ID}", activity.OperationName, activity.Id);
```

Comment:
Debug? Or remove altogether?
```suggestion
                Log.Debug("GetOrAdd for {Name} with ID: {ID}", activity.OperationName, activity.Id);
```

Reply:
Oops, good catch thought I removed my log statements, missed this one - will remove it